CDCgov/prime-reportstream
# Add a gradle task for running ReportStream with the `ETOR_TI_baseurl` set

This PR adds a gradle task for running ReportStream with the `ETOR_TI_baseurl` set. This environment variable is needed when calling the `etorMetadata` endpoints, which are only used for ETOR workflows

Test Steps:
1. Run `./gradlew tiquickrun`
2. Optional - hit the `/api/waters/report/{reportId?/history/etorMetadata` and confirm that you don't get the `Environment variable ETOR_TI_baseurl is not set. Set this variable and run the TI service.` response

## Changes
- adds a gradle task for running ReportStream with the `ETOR_TI_baseurl` set

## Checklist

### Testing

### Process
- [x] Are there licensing issues with any new dependencies introduced? - NA
- [x] Updated the release notes? - NA
- [x] Database changes are submitted as a separate PR? - NA
- [x] DevOps team has been notified if PR requires ops support? - NA

## Linked Issues
- https://github.com/CDCgov/trusted-intermediary/issues/1058

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- NA

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - No
- Does this PR include changes in authentication and/or authorization of existing endpoints?
    - No
- Does this change introduce new dependencies that need vetting?
    - No
- Does this change require changes to our infrastructure?
    - No
- Does logging contain sensitive data?
    - No
- Does this PR include or remove any sensitive information itself?
    - No

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
